Make.config

```diff
@@ -98,18 +98,16 @@ MIN_OSX_BUILD_VERSION=10.14
 MIN_OSX_VERSION_FOR_IOS=10.11
 MIN_OSX_VERSION_FOR_MAC=10.11
 
+# Make sure that the following does match the sdk that is preset in Xcode. Else we will have issues with mtouch (See https://github.com/xamarin/xamarin-macios/issues/7705)
```

Comment:
```suggestion
# Make sure that the following does match the SDK that is present in Xcode. Else we will have issues with mtouch (See https://github.com/xamarin/xamarin-macios/issues/7705)
```

Reply:
Slightly rephrased (just need to warn more IMHO) and otherwise there was a typo in `preset`